src/tokenizer.rs

```diff
@@ -90,23 +91,27 @@ impl Tokenizer {
         k
     }
     pub fn tokenize(&mut self) {
-        #[derive(Clone, Copy)]
         let mut ranges = HashMap::new();
+        let mut block;
         loop {
             let reg = onig::Regex::new(get_regex().as_str()).unwrap();
             let r = reg.find(self.input);
             let capture = reg.captures(self.input).unwrap();
+            println!("{:?}", capture);
+            println!("{:?}", r);
             let mut instruction = HashMap::new();
             if (r.is_none()) || (r.unwrap().0 != self.index) {
                 instruction = self.tokenize_error_context();
                 Tokenization::error_msg("invalid_line", &self.context, &instruction);
             }
             instruction.insert("Index", InstructionValues::Index(self.index));
             instruction.insert("Line", InstructionValues::Index(self.line));
-            let mut block = false;
-            if capture.at(EMPTY_LINE_INDEX) != None {
+            block = false;
+
+            if capture.at(EMPTY_LINE_INDEX).is_some() {
                 instruction.insert("Type", InstructionValues::Type("EMPTY_LINE"));
             } else if capture.at(NAME_OPERATOR_INDEX).is_some() {
+                //println!("Hui");
```

Comment:
Commented code again. Check out the [`env_logger`](https://crates.io/crates/env_logger). It will allow you enable/disable logging any time you want.

Reply:
Done